src/lucky/events/pipe_event.cr

```diff
@@ -0,0 +1,15 @@
+class Lucky::Events::PipeEvent < Pulsar::Event
+  getter :name, :position, :continued
+
+  enum Position
+    Before
+    After
+  end
+
```

Comment:
I'd prob add `after?` and `before?` method just to make it easier to determine if the event was before or after.

Reply:
ah, good point. I like that